### PR TITLE
Deploy on linux using musl

### PR DESCRIPTION
**Pull request by Luni-4**
This PR allows to deploy `Linux` binaries without external dependencies.

Thanks in advance for your review! :)

### CODE REVIEW

**Comment by dwbuiten** [comment]
Is there a way we can verify this works as intended before merging / releasing?

**Comment by Luni-4** [comment]
Yep, you can try the binaries [here](https://github.com/Luni-4/cargo-c/releases/tag/v0.4.9)

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -49,21 +49,59 @@ jobs:
...
+      run: |
+        export OPENSSL_LIB_DIR=$GITHUB_WORKSPACE/lib
+        export OPENSSL_INCLUDE_DIR=$GITHUB_WORKSPACE/include
+        cargo build --target x86_64-unknown-linux-musl --release
+
+    - name: Create zip
+      run: |
+        cd target/x86_64-unknown-linux-musl/release
+        strip cargo-cbuild cargo-cinstall
+        zip $GITHUB_WORKSPACE/cargo-c-linux.zip \
```
Comment (lu-zero) [inline .github/workflows/deploy.yml:92]: make a tar please :)

Comment (Luni-4) [reply .github/workflows/deploy.yml:92]: Is there a particular reason not to use `zip`?

Comment (1480c1) [reply .github/workflows/deploy.yml:92]: tar is better understood on *nix systems and sometimes they do not come with any un-zip programs by default

Comment (Luni-4) [reply .github/workflows/deploy.yml:92]: Seems a good reason, thank you @1480c1 :)

**Comment by dwbuiten** [comment]
Confirmed them, thanks!

**Pull request merged**